tests/desktop_web/test_homepage.py

```diff
@@ -14,16 +16,19 @@ def test_homepage(desktop_web_driver):
     for endpoint in endpoints:
         sentry_sdk.set_tag("endpoint", endpoint)
 
-        # you can filter by se:tda in Sentry's UI
-        endpoint = endpoint + "?se=tda"
-
-        # Randomize the Failure Rate between 1% and 40%
-        n = random.uniform(0.01, .04)
+        for i in range(random.randrange(20)):
```

Comment:
Consider putting the query string generator (including failure rate) in its own function, as it's repeated 3 different times. Not totally necessary now but if we add even one more test that would have to copy the same code, could be useful to make it more reusable.

Reply:
@cstavitsky I'd like to hold off on this re-factor only because the amount of re-testing (locally, staging, Master) I'd have to sit through is quite lengthy. I'm happy to do it on the next. I'm blocking someone else from some SDK upgrade PR's right now.

Reply:
I left a TODO in the code comments.